Source/Core/Core/HW/DSPHLE/UCodes/AXWiiOld.cpp

```diff
@@ -0,0 +1,190 @@
+      addr_lo = m_cmdlist[curr_idx++];
+      SetupProcessing(HILO_TO_32(addr));
+      break;
+
+    case CMD_ADD_TO_LR:
+    case CMD_SUB_TO_LR:
+      addr_hi = m_cmdlist[curr_idx++];
+      addr_lo = m_cmdlist[curr_idx++];
+      AddToLR(HILO_TO_32(addr), cmd == CMD_SUB_TO_LR);
+      break;
+
+    case CMD_ADD_SUB_TO_LR:
+      addr_hi = m_cmdlist[curr_idx++];
+      addr_lo = m_cmdlist[curr_idx++];
+      AddSubToLR(HILO_TO_32(addr));
+      break;
+
+    case CMD_PB_ADDR:
+      addr_hi = m_cmdlist[curr_idx++];
+      addr_lo = m_cmdlist[curr_idx++];
+      pb_addr = HILO_TO_32(addr);
+      break;
+
+    case CMD_PROCESS:
+      ProcessPBList(pb_addr);
+      break;
+
+    case CMD_MIX_AUXA:
+    case CMD_MIX_AUXB:
+    case CMD_MIX_AUXC:
+      volume = m_cmdlist[curr_idx++];
+      addr_hi = m_cmdlist[curr_idx++];
+      addr_lo = m_cmdlist[curr_idx++];
+      addr2_hi = m_cmdlist[curr_idx++];
+      addr2_lo = m_cmdlist[curr_idx++];
+      MixAUXSamples(cmd - CMD_MIX_AUXA, HILO_TO_32(addr), HILO_TO_32(addr2), volume);
+      break;
+
+    case CMD_UPL_AUXA_MIX_LRSC:
+    case CMD_UPL_AUXB_MIX_LRSC:
+    {
+      volume = m_cmdlist[curr_idx++];
+      u32 addresses[6] = {
+          (u32)(m_cmdlist[curr_idx + 0] << 16) | m_cmdlist[curr_idx + 1],
+          (u32)(m_cmdlist[curr_idx + 2] << 16) | m_cmdlist[curr_idx + 3],
+          (u32)(m_cmdlist[curr_idx + 4] << 16) | m_cmdlist[curr_idx + 5],
+          (u32)(m_cmdlist[curr_idx + 6] << 16) | m_cmdlist[curr_idx + 7],
+          (u32)(m_cmdlist[curr_idx + 8] << 16) | m_cmdlist[curr_idx + 9],
+          (u32)(m_cmdlist[curr_idx + 10] << 16) | m_cmdlist[curr_idx + 11],
+      };
+      curr_idx += 12;
+      UploadAUXMixLRSC(cmd == CMD_UPL_AUXB_MIX_LRSC, addresses, volume);
+      break;
+    }
+
+    // TODO(delroth): figure this one out, it's used by almost every
+    // game I've tested so far.
+    case CMD_UNK_0B:
+      curr_idx += 4;
+      break;
+
+    case CMD_OUTPUT:
+    case CMD_OUTPUT_DPL2:
+      addr_hi = m_cmdlist[curr_idx++];
+      addr_lo = m_cmdlist[curr_idx++];
+      addr2_hi = m_cmdlist[curr_idx++];
+      addr2_lo = m_cmdlist[curr_idx++];
+      OutputSamples(HILO_TO_32(addr2), HILO_TO_32(addr), 0x8000, cmd == CMD_OUTPUT_DPL2);
+      break;
+
+    case CMD_WM_OUTPUT:
+    {
+      u32 addresses[4] = {
+          (u32)(m_cmdlist[curr_idx + 0] << 16) | m_cmdlist[curr_idx + 1],
+          (u32)(m_cmdlist[curr_idx + 2] << 16) | m_cmdlist[curr_idx + 3],
+          (u32)(m_cmdlist[curr_idx + 4] << 16) | m_cmdlist[curr_idx + 5],
+          (u32)(m_cmdlist[curr_idx + 6] << 16) | m_cmdlist[curr_idx + 7],
+      };
+      curr_idx += 8;
+      OutputWMSamples(addresses);
+      break;
+    }
+
+    case CMD_END:
+      end = true;
+      break;
+    }
+  }
+}
+
+void AXWiiOldUCode::ProcessPBList(u32 pb_addr)
+{
+  // Samples per millisecond. In theory DSP sampling rate can be changed from
+  // 32KHz to 48KHz, but AX always process at 32KHz.
+  const u32 spms = 32;
```

Comment:
```suggestion
  constexpr u32 spms = 32;
```

Reply:
Suppose I'll change it in the other file too while we're at it.

Reply:
It seems like a good fit, since we're touching the code anyways and it's not a huge refactoring.